go/client/chat_common.go

```diff
@@ -21,7 +40,12 @@ func CheckUserOrTeamName(ctx context.Context, g *libkb.GlobalContext, name strin
 		return &ret, nil
 	}
 
-	// Assume name is a User or a TLF
-	ret := keybase1.UserOrTeamResult_USER
-	return &ret, nil
+	ase, okNet := err.(libkb.AppStatusError)
+	_, okNotFound = err.(teams.TeamDoesNotExistError)
+
+	if !(okNet && keybase1.StatusCode(ase.Code) == keybase1.StatusCode_SCTeamNotFound) && !okNotFound && !strings.HasSuffix(err.Error(), "does not exist") {
```

Comment:
I don't understand why `okNotFound` is false even though that's the correct type of error -- I get an error.errorString instead of the real error type, so I have to check the message manually.